### PR TITLE
Update to latest ed25519 library

### PR DESCRIPTION
**Pull request by fasmat**
This updates the library to use the most recent `spacemeshos/ed25519` library.

DO NOT MERGE BEFORE: https://github.com/spacemeshos/ed25519/pull/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,62 @@
 # ed25519-WASM
-A drop-in replacement for `golang/crypto/ed25519` ([godoc](https://godoc.org/golang.org/x/crypto/ed25519), 
-[github](https://github.com/golang/crypto/tree/master/ed25519))
- with additional functionality compiled to WASM
 
-# Motivation
+A drop-in replacement for `crypto/ed25519` ([pkg.go.dev](https://pkg.go.dev/crypto/ed25519), [github](https://github.com/golang/crypto/tree/master/ed25519))
+with additional functionality compiled to WASM.
```
Comment (fasmat) [inline README.md:4]: This isn't quite right. `spacemeshos/ed25519` is the drop in replacement, but this library only contains the additional functions for singing and verification that create signatures that can be used for key extraction.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        uses: browser-actions/setup-chrome@v1
+      - name: Install wasmbrowsertest
+        run: go install github.com/agnivade/wasmbrowsertest@v0.7.0
+      - name: Setup wasmexec
+        run: mv $(go env GOPATH)/bin/wasmbrowsertest $(go env GOPATH)/bin/go_js_wasm_exec
+
+      # - name: Set up Node.js
+      #   uses: actions/setup-node@v3
+      #   with:
+      #     node-version: ${{ env.node-version }}
```
Comment (fasmat) [inline .github/workflows/ci.yml:35]: TODO: if we don't add jest tests remove this again

File: test/main_test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// Assume add.wasm file exists that contains a single function adding 2 provided arguments
+import { readFileSync } from 'fs';
```
Comment (fasmat) [inline test/main_test.js:2]: TODO: if we don't add jest tests remove this file

**Review by brusherru — APPROVED** [review]
Works well 👍

**Pull request merged**